translator/gobgp.py

```diff
@@ -83,20 +83,29 @@ def _build_path(self, ip, event_data={}):
         as_path.Pack(as_segments)
 
         # Set our community number
-        # global_admin = The Global Administrator field is intended to represent a complete 4-byte ASN
-        # local_data* is for more flexible routing policy. can be set to ME:ACTION:YOU or ASN:Function:Parameter.
-        #
-        # Use local_data2 for any "flag" you want to send to the router that can be used for routing policy decisions.
+        # The ASN gets packed into the community so we need to be careful about size to not overflow the structure
         communities = Any()
-        global_admin = asn
-        local_data1 = community
-        local_data2 = 0
-        large_community = attribute_pb2.LargeCommunity(
-            global_admin=global_admin,
-            local_data1=local_data1,
-            local_data2=local_data2,
-        )
-        communities.Pack(attribute_pb2.LargeCommunitiesAttribute(communities=[large_community]))
+        # Standard community
+        # Since we pack both into the community string we need to make sure they will both fit
+        if asn < 65536 and community < 65536:
```

Comment:
Would 65536 be clearer as 2**16?

Reply:
The integer is less confusing to me, but I'm open to the change if you feel strongly.

Reply:
Nope, that's fine.